ykrt/src/trace/swt/patch.rs

```diff
@@ -0,0 +1,79 @@
+use libc::{mprotect, size_t, sysconf, PROT_EXEC, PROT_READ, PROT_WRITE};
+use std::mem;
+use std::{ffi::c_void, sync::Once};
+
+#[cfg(tracer_swt)]
+use crate::yk_trace_basicblock;
+
+#[cfg(tracer_swt)]
+static ORIGINAL_INSTRUCTIONS_INIT: Once = Once::new();
+#[cfg(tracer_swt)]
+static mut ORIGINAL_INSTRUCTIONS: [u8; 1] = [0; 1];
+#[cfg(tracer_swt)]
+static mut PATCH_INSTRUCTIONS: [u8; 1] = [0xC3];
```

Comment:
Is this a nop?

Reply:
yes, 0xC3 is a return instruction

Reply:
Added comments with info 👉 799644e64d38f56c9447bcac350e86ce0b4e6182

Reply:
I think https://github.com/ykjit/yk/pull/1060/commits/d572cabf3deeef89567a60ca671ac93d08d6743f is meant.